examples/demo/src/products/GridList.tsx

```diff
@@ -33,18 +55,18 @@ const useStyles = makeStyles(theme => ({
     },
 }));
 
-const getColsForWidth = width => {
+const getColsForWidth = (width: Width) => {
     if (width === 'xs') return 2;
     if (width === 'sm') return 3;
     if (width === 'md') return 4;
     if (width === 'lg') return 5;
     return 6;
 };
 
-const times = (nbChildren, fn) =>
+const times = (nbChildren: number, fn: TimesCallback) =>
```

Comment:
We really don't need a named type for the second argument.

Reply:
Done